### PR TITLE
Update RELEASING.md with notes from the last release cycle

### PR DESCRIPTION
**Pull request by pgnickb**
## Merge requirements

Please take into account the following non-code changes that you may need to make with your PR:

- [ ] CHANGELOG entry for user-facing changes
- [ ] Updated the relevant documentation

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -14,14 +14,43 @@ The following are step-by-step instructions for releasing the Promscale extensio
...
 - [ ] Prepare extension release notes, share with team asking for feedback.
-- [ ] Manually tag and push `timescaledev/promscale-extension` docker image (see #354)
 - [ ] Attach release notes to draft release created above.
-- [ ] Publish the GitHub release
+- [ ] Publish the GitHub release.
 
 ## Post-Release
 - [ ] Prepare for the next development cycle:
   - Generate SQL schema and commit it (and upgrade symlinks to it) to the `sql` directory.
+<details>
```
Comment (JamesGuthrie) [inline RELEASING.md:23]: I would prefer it if we would keep the original formatting of this file as a bullet point checklist. If something can't be done as a bullet point item, then we should add some tooling to allow this to be the case.

File: RELEASING.md [hunk 2]
```diff
@@ -14,14 +14,43 @@ The following are step-by-step instructions for releasing the Promscale extensio
...
+<summary>Examples</summary>
+
+Generate SQL file:
+
+```bash
+cargo pgx schema --release
```
Comment (JamesGuthrie) [inline RELEASING.md:29]:
```suggestion
cargo pgx schema --release --out=sql/promscale--$(./extract-extension-version.sh | tr -d '\n').sql
```

File: RELEASING.md [hunk 3]
```diff
@@ -14,14 +14,43 @@ The following are step-by-step instructions for releasing the Promscale extensio
...
+mv sql/promscale-0.5.5.sql sql/promscale--0.5.5.sql
+```
+
+Create symlinks:
+
+```bash
+for i in {0..4}; do ln -s sql/promscale--0.5.5.sql sql/promscale--0.5.${i}--0.5.5.sql; done
```
Comment (JamesGuthrie) [inline RELEASING.md:36]:
```suggestion
./create-upgrade-symlinks.sh
```

File: RELEASING.md [hunk 4]
```diff
@@ -14,14 +14,43 @@ The following are step-by-step instructions for releasing the Promscale extensio
...
+```
+
+Force add the `.sql` files to the commit, as they're in `.gitignore`:
+
+```bash
+git add sql/*--0.5.5*.sql --force
```
Comment (JamesGuthrie) [inline RELEASING.md:42]:
```suggestion
git add --force sql
```

**Review by JamesGuthrie — CHANGES_REQUESTED** [review]

File: RELEASING.md (hunk 4 above)
Comment (pgnickb) [reply RELEASING.md:42]: I'm afraid this will cause unwanted files to be added. Like the `promscale-0.x.x.sql` (the single hyphen) file.

Comment (JamesGuthrie) [reply RELEASING.md:42]: Ya, you're right. As discussed, you can use `extract-extension-version.sh`.

**Comment by sumerman** [comment]
Subsumed by #489

**Pull request closed without merging**